src/payment/data/service.js

```diff
@@ -70,7 +70,6 @@ export function transformResults(data) {
 
   if (results.offers != null) {
     results.offers = results.offers
-      .filter(({ provider }) => provider !== null)
```

Comment:
enterprise offers have a provider, dynamic conditional offers (added for first purchase discount) do not